src/lib/openapi/spec/pat-schema.ts

```diff
@@ -3,27 +3,42 @@ import { FromSchema } from 'json-schema-to-ts';
 export const patSchema = {
     $id: '#/components/schemas/patSchema',
     type: 'object',
+    description:
+        'An overview of a [Personal Access Token](https://docs.getunleash.io/how-to/how-to-create-personal-access-tokens).',
     properties: {
         id: {
-            type: 'number',
+            type: 'integer',
+            description:
+                'The unique identification number for this Personal Access Token. This property is set backend when token is created.',
```

Comment:
It's a little verbose, but is this clearer? You could probably copy this to any of the other descriptions where it's relevant.

```suggestion
                'The unique identification number for this Personal Access Token. This property is set by Unleash when the token is created and cannot be set manually: if you provide a value when creating a PAT, Unleash will ignore it.',
```

Reply:
I picked the other one you suggested! Thanks